.github/workflows/checks.yml

```diff
@@ -24,3 +24,27 @@ jobs:
 
       - name: Dependency audit
         run: ./run_audit.sh
+
+
+  publish:
```

Comment:
this is prob a dumb question but what are we using for our CI? Like which service is picking up and executing this file?